pkg/cmd/metadata.go

```diff
@@ -18,6 +18,14 @@ func newMetadataCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "metadata",
 		Short: "Run a mock metadata service",
+		Long: `
+This subcommand starts a mock metadata service which
+responds to queries from the Relay client SDKs, to help debug
+and test step entrypoint code in your local environment. You can 
```

Comment:
Minor point, but I would replace (and avoid the use of) "entrypoint" with execution or similar language.